lib/streamlit/elements/button.py

```diff
@@ -112,6 +112,20 @@ def download_button(
     ) -> bool:
         """Display a download button widget.
 
+        Download button has a few constraints:
+
+        - Download button is designed to download data that is stored in the
```

Comment:
Does the `-` symbol have any special meaning in a docstring markup language, that can break the structure?

Reply:
@kajarenc Not that I know of. I used `-` to create the bulleted list. You can see how it is rendered in the [docs here](https://streamlit-streamlit--3752.com.readthedocs.build/en/3752/api.html#streamlit.download_button) 
If you think they don't look good in the docstring, I can remove  them so that the three bullet points are separate paragraphs instead.